screens/Onboarding/OnboardingScreen.tsx

```diff
@@ -1,10 +1,70 @@
-import { View, Text } from "react-native"
-import { styles } from "./styles"
+/* eslint-disable react/no-unescaped-entities */
+//this is the login screenx
+import React from 'react';
+import {
+  Text,
+  View,
+  TextInput,
+  TouchableOpacity,
+  Image
+} from 'react-native';
 
-export const OnboardingScreen = () => {
+import styles from './styles'; // Make sure the path is correct
+
+
+const LoginScreen: React.FC = () => {
+  
   return (
     <View style={styles.container}>
-      <Text>This is the onboarding screen</Text>
+        
+        <Image source={require("../../assets/icon.png")} style={styles.image} />
+      
+    
+
+      {/* Username/Email Field */}
+      <TextInput
+        style={styles.inputField}
+        placeholder="Username or email"
+        placeholderTextColor={'black'}
+        keyboardType="email-address"
+        autoCapitalize="none"
+      />
+
+      {/* Password Field */}
+      <TextInput
+        style={styles.inputField}
+        placeholder="Password"
+        placeholderTextColor={'black'}
+        secureTextEntry
+        autoCapitalize="none"
+      />
+
+
+      {/* Log In Button */}
+      <TouchableOpacity style={styles.button}>
+        <Text style={styles.buttonText}>Log In</Text>
+      </TouchableOpacity>
+
+      <View >
+        <Text style={styles.smallText}>Forgot password?</Text>  
+
+      </View>
+
+      {/* Continue with Google */}
+      <TouchableOpacity style={styles.buttonGoogle}>
+        <Image 
```

Comment:
Please don't use a button as an image. It takes much longer to render an image instead of a styled view. it is more preferable to use an icon or a small image for the logo itself.